### ISSUE TITLE
WV_SLC high noise problem on ipf 2.9

### ISSUE THREAD

**Issue by oarcher** [umr-lops/xsar#61]
Time serie show negative denoised sigma0 for ipf 2.9:
![image](https://user-images.githubusercontent.com/4264660/158863840-dac9afb2-e4f8-4999-84d9-dbab5d2f1d4b.png)

Expected to see sigma0 positive and close to zero, and plot show values around -10

**Comment by oarcher** [comment]
opening the safe with `luts=True` give the `noise_lut_range` variable:



```python
xds = xsar.Sentinel1Dataset(
  'SENTINEL1_DS:S1B_WV_SLC__1SSV_20180323T035038_20180323T042239_010153_012716_226F.SAFE:WV_093',  
  luts=True
)
```
![image](https://user-images.githubusercontent.com/4264660/158868295-36b7e2f4-743a-4676-bff5-e6ef851e939b.png)


The nesz is very high (around 10). its should be less than sigma0, ie in the (0,0.4) range.

**Comment by agrouaze** [comment]
MPC Sentinel-1 group proposed this patch in 2020:
```python
noise_calibration_cst_pp1 = {'S1A':
                                 {'wv1':{'EAPold':38.13,'EAPnew':38.13},
                                  'wv2':{'EAPold':36.84,'EAPnew':36.84} 
                                },
                             'S1B':
                                 {'wv1' : {'EAPold' : 39.30,'EAPnew' : 39.30},
                                  'wv2' : {'EAPold' : 37.44,'EAPnew' :37.44} 
                             } 
                             } 
```

see ticket  https://jira-projects.cls.fr/browse/MPCS-2007

**Comment by oarcher** [comment]
closed by #64 . Noise is patched by default, and can be disabled with `patch_variable=False` from `Sentinel1Dataset`.
